src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -222,9 +256,25 @@ export default {
                 });
         },
         timeSystemChange(system) {
+            let timeSystem = this.openmct.time.timeSystem();
             // reset timesystem dependent variables
             this.timeKey = system.key;
             this.timeFormat = this.openmct.telemetry.getValueFormatter(this.metadata.value(this.timeKey));
+            this.durationFormatter = this.getFormatter(timeSystem.durationFormat || DEFAULT_DURATION_FORMATTER);
+        },
+        clockChange(clock) {
+            this.clock = clock;
```

Comment:
We should only attempt this is `openmct.time.timeSystem().isUTCBased` is `true`. We don't necessarily know the units of time for non-utc based time systems. It's an edge case, but one that we should address.

Reply:
Agreed, if it isn't UTC based we'll short circuit.